integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,33 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: job
+    selector:
+      query:  "true"
+    port:
+      entity:
+        mappings:
+          identifier: 'if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1]'
+          title: '(.data.displayName // .displayName)'
+          blueprint: '"job"'
+          properties:
+            jobName: '.data.fullName // .fullName'
+            url: '.url | if test("://") then sub("^[^/]+//[^/]+"; "") else . end'
+            jobStatus: '.type | split(".") | last'
+            timestamp: .time
+  - kind: build
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: '(.data.fullDisplayName // .fullDisplayName) | sub(" "; "-"; "g") | sub("#"; ""; "g")'
+          title: '.data.displayName // .displayName'
+          blueprint: '"build"'
+          properties:
+            buildStatus: '.data.result // .result'
+            buildUrl: '.url | if test("://") then sub("^[^/]+//[^/]+"; "") else . end'
+            buildDuration: '.data.duration // .duration'
+            timestamp: '(.time // .timestamp) as $input | if $input | type == "number" then ($input / 1000 | todate) else $input end'
+          relations:
+            job: '(.source // .url) as $input | if $input | contains("://") then ($input | split("://")[1] | sub("^.*?/"; "")) else $input end | tostring | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1] | sub("-[0-9]+$"; "")'
```

Comment:
The build dont hold the job id?

if not then lets talk about how to ingest custom information to the data